pipelines/deploy-fbc-operator/0.1/deploy-fbc-operator.yaml

```diff
@@ -312,7 +312,7 @@ spec:
 
               # Look up the *original* image for which this is a mirror
               upstream_image=$(jq -r --arg mirror "${reg_and_repo}" '
-                to_entries[] | select(.value[] == $mirror) | .key' /tekton/home/mirror-map.txt)
+                to_entries[] | select(.key == $mirror) | .key' /tekton/home/mirror-map.txt)
```

Comment:
I see what the issue is - the mirror lookup logic is reversed. Even with this change, it still won’t work correctly. If $reg_and_repo is registry.redhat.io/openshift-lightspeed/lightspeed-operator-bundle (which is the "key" in the mirror map), jq will simply return the same string. I think I know how to fix it. I'll try it on my end.